src/main/java/com/conveyal/r5/point_to_point/PointToPointRouterServer.java

```diff
@@ -1048,4 +1121,43 @@ private static float roundSpeed(float speed) {
         return Math.round(speed * 1000) / 1000;
     }
 
+    private static AnalystClusterRequest makeClusterRequest(TransportNetwork transportNetwork,
```

Comment:
The name of this function should probably indicate that this request serves a specific purpose. Even if its purpose within the PR or the commit is clear, as an element within PointToPointRouterServer.java it's not clear that this is intended only for generating isochrones.